File: eBeertijaBackend/eBeertijaBackend/DTOs/RacunIzvjestajDto.cs
using ebeertijaBackend.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ebeertijaBackend.DTOs
{
    public class RacunIzvjestajDto
    {
        public int Id { get; set; }
        public int Broj { get; set; }
        public string BrojOpis { get; set; }
        public DateTime Datum { get; set; }
        public int? NarudzbaId { get; set; }
        public NarudzbaDto Narudzba { get; set; }
        public double UkupnoSaPdvom { get; set; }
        public double UkupnoBezPdva { get; set; }
        public double IznosPdv { get; set; }
        public int StolId { get; set; }
        public string OznakaStola { get; set; }
        public int UserId { get; set; }
        public bool IsStorniran { get; set; }
        public int? StorniraniRacunId { get; set; }
        public string BrojStorniranogRacuna { get; set; }
        public string NazivKorisnika { get; set; }
        public ICollection<StavkaRacunaDto> StavkeRacuna { get; set; }
    }
}

File: eBeertijaFrontend/src/app/Models/RacunIzvjestajDto.ts
import { NarudzbaDto } from './NarudzbaDto';
import { StavkaRacunaDto } from './StavkaRacunaDto';

export class RacunIzvjestajDto {
    id?: number;
    broj?: number;
    brojOpis?: string;
    datum?: Date;
    narudzbaId?: number;
    narudzba?: NarudzbaDto;
    ukupnoSaPdvom?: number;
    ukupnoBezPdva?: number;
    iznosPdv?: number;
    stolId?: number;
    oznakaStola?: string;
    userId?: number;
    isStorniran?: boolean;
    storniraniRacunId?: number;
    brojStorniranogRacuna?: string;
    nazivKorisnika?: string;
    stavkeRacuna?: StavkaRacunaDto[];
}
